services/filemanager/filemanager.go

```diff
@@ -63,6 +63,10 @@ func init() {
 // New returns FileManager backed by configured provider
 func (*FileManagerFactoryT) New(settings *SettingsT) (FileManager, error) {
 	switch settings.Provider {
+	case "S3_DATALAKE":
```

Comment:
Why do we need a duplicate fileMangaer for S3 Datalake when we can use S3?

Reply:
The provider comes in from above and it has S3_DATALAKE in it hardcoded. Let me see what change can I make !